openMediaAppearIn/openMediaAppearIn.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="http://www.w3schools.com/lib/w3.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
+    <script src="../common/micc.js"></script>
+    <script src="openMediaVideoConferencing.js"></script>
```

Comment:
This file name no longer matches the js file name.  Retest application to ensure in good working order since changes.